.github/pull_request_template.md

```diff
@@ -1,14 +1,60 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: (TODO link CONTRIBUTING.md) and developer guide (TODO link dev guide)
+2. Make sure the to check these marks:
```

Comment:
reviewers probably want to see the checkboxes, I would not put this inside a comment